### PR TITLE
Added company name to Window binaries

### PR DESCRIPTION
**Pull request by CryptoNinjaGeek**
### Scope & Purpose

Backport of [20027](https://github.com/arangodb/arangodb/pull/20027)

- [X] :hankey: Bugfix
- [X] :pizza: New feature
- [ ] :fire: Performance improvement
- [ ] :hammer: Refactoring/simplification

### Checklist

- [ ] Tests
  - [ ] **Regression tests**
  - [ ] C++ **Unit tests**
  - [ ] **integration tests**
  - [ ] **resilience tests**
- [ ] :book: CHANGELOG entry made
- [ ] :books: documentation written (release notes, API changes, ...)
- [ ] Backports
  - [ ] Backport for 3.11: *(Please link PR)*
  - [ ] Backport for 3.10: *(Please link PR)*

#### Related Information

*(Please reference tickets / specification / other PRs etc)*

- [ ] Docs PR: 
- [ ] Enterprise PR:
- [ ] GitHub issue / Jira ticket:
- [ ] Design document:

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -236,6 +236,7 @@ set(ARANGO_EXPORT_FRIENDLY_STRING  "arangoexport - data exporter")
 set(ARANGO_IMPORT_FRIENDLY_STRING  "arangoimport - data importer")
 set(ARANGOSH_FRIENDLY_STRING       "arangosh - commandline client")
 set(ARANGO_VPACK_FRIENDLY_STRING   "arangovpack - VelocyPack formatter")
```
Comment (dothebart) [inline CMakeLists.txt:238]: DRY please. ARANGODB_PACKAGE_VENDOR can be used.

**Review by dothebart — COMMENTED** [review]
please re-use the pre-existing `ARANGODB_PACKAGE_VENDOR` to fill `ARANGODB_PACKAGE_COMPANY` variable in the main cmake file. 

You only intend to do that to the .exe s themselves, not the installer, right?
else LGTM.

**Review by dothebart — APPROVED** [review]

File: CHANGELOG [hunk 2]
```diff
@@ -1,6 +1,9 @@
 v3.11.5 (XXXX-XX-XX)
 --------------------
 
+
+* MDS-1164: Added Publisher and Company to the Window binaries
```
Comment (KVS85) [inline CHANGELOG:5]: 's/Windows/Windows'.

**Review by KVS85 — COMMENTED** [review]
Would second https://github.com/arangodb/arangodb/pull/20029#pullrequestreview-1717871027.

File: CHANGELOG (hunk 2 above)
Comment (CryptoNinjaGeek) [reply CHANGELOG]: Done

File: CMakeLists.txt (hunk 1 above)
Comment (CryptoNinjaGeek) [reply CMakeLists.txt]: Done

**Comment by CryptoNinjaGeek** [comment]
https://jenkins.arangodb.biz/job/arangodb-ANY-windows-packages/4741/

**Review by KVS85 — APPROVED** [review]

**Pull request merged**